packages/blocks/src/store/process-block-type.js

```diff
@@ -102,13 +130,21 @@ export const processBlockType =
 			),
 		};
 
+		blockType.supports = stabilizeBorderSupports( blockType.supports );
```

Comment:
We'll need some unit tests to cover all the stabilization, similar to those in https://github.com/WordPress/gutenberg/pull/63401.